pkg/simple/client/multicluster/options.go

```diff
@@ -79,5 +79,5 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, s *Options) {
 		"This field is used when generating deployment yaml for agent.")
 
 	fs.DurationVar(&o.ClusterControllerResyncSecond, "cluster-controller-resync-second", s.ClusterControllerResyncSecond,
-		"Cluster controller resync second to sync cluster resource.")
+		"Cluster controller resync second to sync cluster resource. e.g. 30s 60s 120s...")
```

Comment:
better to start `2m, 5m, 10m`, small resync period increases load

Reply:
Agree with that. I will update the comment.